pkg/collector/check/check.go

```diff
@@ -56,6 +56,8 @@ type Check interface {
 	GetDiagnoses() ([]diagnosis.Diagnosis, error)
 	// IsHAEnabled returns if High Availability is enabled for this check
 	IsHAEnabled() bool
+	// IsHASupported returns if the check is compatible with High Availability
+	IsHASupported() bool
```

Comment:
`CheckWrapper#IsHASupported` is the only place where this interface method is called. Since we don't call anywhere useful, this can be safely removed.